buildx/working-with-buildx.md

```diff
@@ -14,24 +14,42 @@ multiple nodes concurrently.
 
 ## Install
 
-Docker Buildx is included in Docker Desktop and Docker Linux packages when
-installed using the [DEB or RPM packages](../engine/install/index.md).
+### Windows and macOS
 
-You can also download the latest `buildx` binary from the
-[Docker buildx](https://github.com/docker/buildx/releases/latest){:target="_blank" rel="noopener" class="_"} releases page
-on GitHub, copy it to `~/.docker/cli-plugins` folder with name
-`docker-buildx` and change the permission to execute:
+Docker Buildx is included in [Docker Desktop](https://www.docker.com/products/docker-desktop){:target="_blank" rel="noopener" class="_"}
+for Windows and macOS.
 
-```console
-$ chmod a+x ~/.docker/cli-plugins/docker-buildx
+### Linux packages
+
+Docker Linux packages also include Docker Buildx when installed using the
+[DEB or RPM packages](../engine/install/index.md).
+
+### Manual download
+
+You can also download the latest binary from the [releases page on GitHub](https://github.com/docker/buildx/releases/latest){:target="_blank" rel="noopener" class="_"}.
+
+Rename the relevant binary for your OS to `docker-buildx` and copy it to
+`$HOME/.docker/cli-plugins`.
```

Comment:
I guess if users use `sudo` to run `docker`, they should pick one of the instructions below, correct?

We also may need to add some information about which path to pick in general, as they're processed in a specific order, which means that users could inadvertently "mask" the version that comes with their install with an older version (and never update)

Reply:
> I guess if users use sudo to run docker, they should pick one of the instructions below, correct?

Correct. We might stipulate that this is for an advanced usage and like you said warn users about it. Maybe add a warning bloc about it, wdyt?